bld/namelist_files/namelist_defaults_ctsm.xml

```diff
@@ -1099,6 +1129,19 @@ lnd/clm2/surfdata_map/release-clm5.0.18/surfdata_1x1_numaIA_hist_78pfts_CMIP6_si
 <fsurdat hgrid="1x1_smallvilleIA" sim_year="2000" use_crop=".true." >
 lnd/clm2/surfdata_map/ctsm1.0.dev094-2-g633be0eb/surfdata_1x1_smallvilleIA_hist_78pfts_CMIP6_simyr2000_c200521.nc</fsurdat>
 
+<fsurdat hgrid="mpasa480"  sim_year="2000">
+lnd/clm2/surfdata_map/surfdata_mpasa480_hist_78pfts_CMIP6_simyr2000_c211110.nc </fsurdat>
```

Comment:
@jtruesdal these files should be moved to the ctsm5.1.dev052 subdirectory.

- [ ] Move surfacedatasets to the lnd/clm2/ctsm5.1.dev052 subdirectory.

Reply:
@ekluzek just getting back to this and would like to do what I can this week to move this forward.  Given the time that has elapsed are there updates to the directory that these datasets should be moved to. I'll update the branch with your recommendations.

Reply:
@ekluzek I have a draft atmosphere PR in now that will need these datasets.   I updated the code with the ctsm5.1.dev052 subdirectory change.  I do not have permissions to move these files on glade though.

Reply:
OK, I fixed the permission issue, so you should be able to move them in.

Reply:
All the files have been moved.  Let me know when you would like me to rimport them.

Reply:
@ekluzek after moving the files into place I made a 1 day mpasa480 F2000climo test which accessed the new surfdat file and ran to completion.  Please let me know when it is OK to rimport these new datasets into the data repo.

Reply:
@jtruesdal you should run rimport on most of these files at this point. I suggest to NOT import the landuse.timeseries for mpasa15 as it's so large. It could just be kept on glade for running on cheyenne, so it would be available but not everywhere. If we rimport it, it gets added to our svn archives and other CESM inputdata archival places, and it's not clear to me that it's going to be useful. Running very many years of mpas15 would be extremely expensive, it seems unlikely that someone is going to run all the way from 1850 to 2100 for mpasa15.